src/makeCalculator.js

```diff
@@ -38,6 +38,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, value) {
+      this.result = operation(this.result, value);
+
+      return this;
+    },
```

Comment:
```suggestion
    operate(operation, value) {
      operation(value);

      return this;
    },
```
lets make work like this